src/revivalpmmp/pureentities/entity/monster/walking/Wolf.php

```diff
@@ -407,13 +412,13 @@ public function tame(Player $player): bool {
      *
      * @param bool $tamed
      */
-    public function setTamed(bool $tamed) {
-        if ($tamed) {
+    public function setTamed(bool $option) {
```

Comment:
What was the reason behind changing this?

Reply:
again, ask @95CivicSi as this was part of the master merge that I assumed was an update that should be preserved.